### DISCUSSION TITLE
New drinkables apply the full effect with only a sip

### DISCUSSION THREAD

**Discussion by xolophreny** [FakeFishGames/Barotrauma#13827] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Ethanol and Rum apply the full extent of their effects when only clicking the use button shortly and using a few percent of the item

### Reproduction steps

1. Sub editor with a supply of Ethanol
2. Hold the Ethanol, slightly sniff the Ethanol with the aim button
3. Feel as if you'd drunk the whole thing

https://github.com/FakeFishGames/Barotrauma/assets/41170402/1393408a-80af-4e6d-b141-41a422861407

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

Sub editor

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Linux

**Comment by Regalis11** [comment]
Thank you for the report! I [created a ticket about this in the issue tracker]([New drinkables apply the full effect with only a sip](https://github.com/FakeFishGames/Barotrauma/issues/13829#top)), we'll look into it!

**Comment by JeroonBosch** [comment]
Issue has been addressed internally, and will be making it to the live version shortly, thanks for your report - closing this now.
